### ISSUE TITLE
Unspecified help for main command results in an "unitialized value" error

### ISSUE THREAD

**Issue by djerius** [polettix/App-Easer#6]
Not specifying help at the top level is definitely a foot-gun, but perhaps a message indicating that's what amiss?

```
use App::Easer V2 => 'run';
exit run( {  execute => sub  { return $_[0]->run_help }, },
  $0, @ARGV
);
```
results in

```
Use of uninitialized value $stuff[0] in join or string at [...]/App/Easer/V2.pm line 882.

This command has no option
No sub-commands
```

Thanks!

**Comment by polettix** [comment]
Fixed in 4c4a875, will be in next release. Thanks!

**Comment by polettix** [comment]
[Release v2.008 is out](https://metacpan.org/release/POLETTIX/App-Easer-2.008)
